src/index.json

```diff
@@ -41961,6 +41961,60 @@
                     "version": "0.1.2"
                 },
                 "sha256Digest": "a2e4ef513b74c138c3c445c6c494eb9e4677338f53e1b5a634a280b57bd70fc5"
+            },
+            {
+                "downloadUrl": "https://github.com/Azure/partnercenter-cli-extension/releases/download/v0.2.3-alpha/partnercenter-0.2.3-py3-none-any.whl",
+                "filename": "partnercenter-0.2.3-py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.67",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/partnercenter-cli-extension/tree/main/partnercenter"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "partnercenter",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                "azure-storage-blob",
+                                "docker",
+                                "pydantic",
```

Comment:
Hi @kevinhillinger, please ping the version of "pydantic", it has issues in the recent versions. https://github.com/pydantic/pydantic/issues/6361#issuecomment-1620842286